tasks/ca/statcan/geo.py

```diff
@@ -13,22 +13,25 @@
 GEO_CD = 'cd_'
 GEO_CSD = 'csd'
 GEO_CMA = 'cma'
+GEO_DA = 'da'
```

Comment:
To avoid breaking all the other geographies we should define it as `GEO_DA = 'da_'`